src/login/logind-action.c

```diff
@@ -346,6 +390,16 @@ int manager_handle_action(
                 return 1;
         }
 
+        if (handle == HANDLE_SECURE_ATTENTION_KEY) {
+                if (!is_edge)
+                        return 0;
+
+                if (!action_seat)
+                        return 0;
+
```

Comment:
you have the exact same condition checks as first thing in manager_handle_action_secure_attention_key(), hence you can drop them here.